lib/vsc/mympirun/rm/__init__.py

```diff
@@ -1,27 +1,30 @@
-##
-# Copyright 2009-2012 Ghent University
-# Copyright 2009-2012 Stijn De Weirdt
 #
-# This file is part of VSC-tools,
+# Copyright 2009-2016 Ghent University
+#
+# This file is part of vsc-mympirun,
 # originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
 # with support of Ghent University (http://ugent.be/hpc),
-# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
-# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
 # and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
 #
-# http://github.com/hpcugent/VSC-tools
+# https://github.com/hpcugent/vsc-mympirun
 #
-# VSC-tools is free software: you can redistribute it and/or modify
+# vsc-mympirun is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation v2.
 #
-# VSC-tools is distributed in the hope that it will be useful,
+# vsc-mympirun is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with VSC-tools. If not, see <http://www.gnu.org/licenses/>.
-##
+# along with vsc-mympirun.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Allow other packages to extend this namespace, zip safe setuptools style
+"""
+import pkg_resources
+pkg_resources.declare_namespace(__name__)
 
-import local, pbs
```

Comment:
this will probably break the current code, yes. but the `__init__` should be empty, it's up to jeroen to fix this